include/libdnf5/base/transaction.hpp

```diff
@@ -30,6 +30,7 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include "libdnf5/rpm/transaction_callbacks.hpp"
 
 #include <optional>
+#include <string_view>
```

Comment:
Why is the `string_view` file included to the public header?
It is not used in this header. Please move the line to the `transaction_impl.hpp` file.

Reply:
Thanks, I will fix it.